### DISCUSSION TITLE
Invalid referring URL for many features

### DISCUSSION THREAD

**Discussion by AlbaVore** [craigk5n/webcalendar#519] [Q&A] [answered]
Hello, I have version 1.9.1 installed. I am trying to update and admin the calendar but am getting a lot of 'Invalid referring URL' links when trying to access features such as the system settings, preferences as well as when editing calendar entries.
The site is using php v7.4.3. When I upgrade to v8 then the calendar doesn't load.

Would appreciate some input as theres a bit of time and effort invested in the project which is here:
https://thegorbals.co.uk/thegorbalscalendar/month.php

**Comment by bbannon** [comment]
I think 'include functions.php' is being called late.
I also think there is a PR to fix it, but it hasn't been approved yet.

We're all volunteers and sometimes real life gets in the way of WebCalendar.

**Reply by AlbaVore** [reply]
Thanks for your input. Is this a common issue?
I appreciate your time.

**Comment by bbannon** [comment]
Well, Craig is just 1 guy. He is the one to approve PRs. He has other things to do. 
 
We could always use more help submitting PRs when someone finds a problem.

**Reply by AlbaVore** [reply]
Hello again Bruce, I really appreciate that. Craig has done a great job with webcalendar. The site I am working on is one I am doing as a volunteer for my local community.
Sorry for the maybe obvious question but what is a PR so that I am clear? I am afraid I maybe don't have the coding expertise to contribute much by way of resolving issues but I can certainly report issues.

**Reply by bbannon** [reply]
PR = Pull Request
The way to submit fixes or new features to repositories on GitHub.

I often feel the same way about them. "Am I good enough?"
But I do it anyway. :) Mostly just for fun.

**Reply by AlbaVore** [reply, marked as answer]
Hello again.
This worked:
[https://github.com/craigk5n/webcalendar/issues/69#issuecomment-1468716408](url)

**Comment by AlbaVore** [comment]
Issue resolved.
